front/app/components/CityLogoSection/index.tsx

```diff
@@ -49,40 +50,50 @@ const TenantLogo = injectIntl(
 
 interface Props {}
 
-const CityLogoSection = memo(
-  ({ intl: { formatMessage } }: Props & InjectedIntlProps) => {
-    const locale = useLocale();
-    const appConfiguration = useAppConfiguration();
+const CityLogoSection = ({
+  localize,
+  intl: { formatMessage },
+}: Props & InjectedIntlProps & InjectedLocalized) => {
+  const locale = useLocale();
+  const appConfiguration = useAppConfiguration();
 
-    if (!isNilOrError(appConfiguration)) {
-      const currentTenantLogo =
-        appConfiguration.data.attributes.logo?.medium || null;
-      const tenantSite =
-        appConfiguration.data.attributes.settings.core.organization_site;
-      const footerLocale = `footer-city-logo-${locale}`;
+  if (!isNilOrError(appConfiguration)) {
+    const currentTenantLogo =
+      appConfiguration.data.attributes.logo?.medium || null;
+    const tenantSite =
+      appConfiguration.data.attributes.settings.core.organization_site;
+    const footerLocale = `footer-city-logo-${locale}`;
+    const localizedOrgName = localize(
+      appConfiguration.data.attributes.settings.core.organization_name
+    );
 
-      if (currentTenantLogo) {
-        return (
-          <Fragment
-            title={formatMessage(messages.iframeTitle)}
-            name={footerLocale}
-          >
-            <Container id="hook-footer-logo">
-              {tenantSite ? (
-                <LogoLink href={tenantSite} target="_blank">
-                  <TenantLogo src={currentTenantLogo} />
-                </LogoLink>
-              ) : (
-                <TenantLogo src={currentTenantLogo} />
-              )}
-            </Container>
-          </Fragment>
-        );
-      }
+    if (currentTenantLogo) {
+      return (
+        <Fragment
+          title={formatMessage(messages.iframeTitle)}
+          name={footerLocale}
+        >
+          <Container id="hook-footer-logo">
+            {tenantSite ? (
+              <LogoLink href={tenantSite} target="_blank">
+                <TenantLogo
+                  src={currentTenantLogo}
+                  localizedOrgName={localizedOrgName}
+                />
+              </LogoLink>
+            ) : (
+              <TenantLogo
+                localizedOrgName={localizedOrgName}
+                src={currentTenantLogo}
+              />
+            )}
+          </Container>
+        </Fragment>
+      );
     }
-
-    return null;
   }
-);
 
-export default injectIntl(CityLogoSection);
+  return null;
+};
+
+export default injectLocalize(injectIntl(CityLogoSection));
```

Comment:
We're trying to move from HOCs to hooks and there's a hook for localize (useLocalize). Let's use that one instead?

Reply:
The same for the other components. :)